apps/passport-client/components/screens/LoginScreens/LoginScreen.tsx

```diff
@@ -166,7 +257,23 @@ export function LoginScreen(): JSX.Element {
         </>
       )}
 
-      {!state.loggingOut && (
+      {tryStorageAccess && (
+        <TextCenter>
+          <Spacer h={24} />
+          Do you want to connect to Zupass?
```

Comment:
This text should really be more specific about the ZApp in which the user is connecting.
At minimum it seems like it should say something about the URL of window.parent so the user sees what they're connecting to Zupass.
Out of the scope of this PR, this would be a place to plug into more information (like a known app or developer name) from a registry of approved Zapps if we had one.  That would come after any security check to verify that the Zapp should be allowed to connect at all.  I think we do need at least some form of that (like a whitelist of hostnames) before ZApps go to prod for all users, so that we ensure someone can't create a phishing site which embeds Zupass in an iframe and steals all a user's PODs.

Reply:
I've added specific text explaining which Zapp is connecting.